packages/core/admin/ee/server/services/review-workflows/stages.js

```diff
@@ -38,7 +38,7 @@ module.exports = ({ strapi }) => {
         )
       );
 
-      metrics.sendDidCreateStage();
+      metrics.sendDidCreateStage(totalNumberOfStages);
```

Comment:
minor thing, I find it a bit out of place to have a custom totalNumberOfStages parameter, could we do something like this:

what if we had some sort of ctx attribute?
```
    async createMany(stagesList, { fields } = {}, { totalNumberOfStages }) {
      const params = { select: fields ?? '*' };

      const stages = await Promise.all(
        stagesList.map((stage) =>
          strapi.entityService.create(STAGE_MODEL_UID, { data: stage, ...params })
        )
      );

      metrics.sendDidCreateStage(totalNumberOfStages);
```